odoo/addons/base/models/res_country.py

```diff
@@ -96,6 +96,11 @@ def _name_search(self, name='', args=None, operator='ilike', limit=100, name_get
 
         return ids
 
+    @api.model
+    @tools.ormcache('code')
+    def _country_get_phone_code(self, code):
```

Comment:
the prefix seems a bit redundant considering this is a method of `res.country`?

```suggestion
    def _phone_code_for(self, code):
```

which you'll be calling as:

```py
request.env['res.country']._phone_code_for(country_code)
```